File: MAMKSchedule/src/main/java/ru/shika/mamkschedule/mamkschedule/Lesson.java
package ru.shika.mamkschedule.mamkschedule;

import android.graphics.drawable.Drawable;

public class Lesson
{
	String start, end, room, name, teacher;

	public Lesson(String start, String end, String room, String name, String teacher)
	{
		this.start = start;
		this.end = end;
		this.room = room;
		this.name = name;
		this.teacher = teacher;
	}

	public static class DrawerItem
	{
		String string;
		Drawable drawable;

		public DrawerItem(String string, Drawable drawable)
		{
			this.string = string;
			this.drawable = drawable;
		}
	}
}

File: schedule/src/main/kotlin/ru/shika/app/main/data/model/Lesson.kt
package ru.shika.app.main.data.model

import io.realm.RealmList
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

/**
 * Model for one lesson
 */
open class Lesson(
    @PrimaryKey
    var id       : Long               = 0,
    var courseId : String             = "",
    var name     : String             = "",
    var group    : String             = "",
    var teachers : RealmList<Teacher> = RealmList(),
    var rooms    : RealmList<Room> = RealmList(),
    var start    : Date = Date(),
    var end      : Date = Date()
) : RealmObject()
